adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -33,26 +34,80 @@ - (void)dealloc {
   [self stopBeingDelegate];
 }
 
+#pragma mark - GAD Ad Network Protocol Banner Methods (MREC)
+
 - (void)getBannerWithSize:(GADAdSize)adSize {
-  NSError *error = [NSError
-      errorWithDomain:kGADMAdapterVungleErrorDomain
-                 code:0
-             userInfo:@{NSLocalizedDescriptionKey : @"Vungle doesn't support banner ads."}];
-  [_connector adapter:self didFailAd:error];
-}
+  self.adapterAdType = MREC;
 
-- (void)loadInterstitialAd {
-  NSError *error = [[VungleRouter sharedInstance] loadAd:desiredPlacement withDelegate:self];
-  if (error) {
-    [_connector adapter:self didFailAd:error];
+  // Check if given banner size is in MREC.
+  if (!CGSizeEqualToSize(adSize.size, kGADAdSizeMediumRectangle.size)) {
+    NSError *error = [NSError
+        errorWithDomain:kGADMAdapterVungleErrorDomain
+                   code:0
+               userInfo:@{
+                 NSLocalizedDescriptionKey : @"Vungle only supports banner ad size in 300 x 250."
+               }];
+    [self.connector adapter:self didFailAd:error];
```

Comment:
Please do not use `self` when applicable. Use `_` syntax (e.g. `_connector`)

Reply:
@joshuajplagonera In Vungle iOS team coding conventions, we try to use `self` instead of underscore, outside of initialization functions. Using underscore is Google's coding convention? If so, we need update all code using `self` in adapter codes? Thanks.

Reply:
Hi @ashinagawa , in that case, then it is fine to use `self` in the meantime.

Reply:
@joshuajplagonera OK!
If this is Google's required Code Standard, please let us know. We will update all classes ti follow it in next version.
Thanks!